src/features/comments.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-param-reassign */
+
+import {
+  PayloadAction,
+  Slice,
+  createAsyncThunk,
+  createSlice,
+} from '@reduxjs/toolkit';
+import { Comment } from '../types/Comment';
+import { getPostComments } from '../api/comments';
+import * as commentsApi from '../api/comments';
+
+export type CommentsState = {
+  comments: Comment[];
+  loaded: boolean;
+  hasError: boolean;
+};
+
+const initialState: CommentsState = {
+  comments: [],
+  loaded: true,
+  hasError: false,
+};
+
+export const loadComments = createAsyncThunk(
+  'comments/fetch',
+  (postId: number) => {
+    return getPostComments(postId);
+  },
+);
+
+export const commentsSlice: Slice<CommentsState> = createSlice({
+  name: 'comments',
+  initialState,
+  reducers: {
+    set: (state, action: PayloadAction<Comment>) => {
+      state.comments.push(action.payload);
+    },
+    deleteCom: (state, action: PayloadAction<number>) => {
```

Comment:
do not use short versions - `deleteComment` is definitely better
Or just `delete` in case you don't want to repeat the slice name in each action name. But such repeating might be useful when you import it somewhere as separate action